bot.py

```diff
@@ -0,0 +1,300 @@
+"""Главный файл приложения бота."""
+
+import datetime
+import logging
+import os
+import re
+import sys
+import time
+import pytz
+import requests
+import configparser
+import locale
+
+
+from dotenv import load_dotenv
+from http import HTTPStatus
+from logging.handlers import RotatingFileHandler
+from mycustomerror import MyCustomError
+from telegram import Bot, ReplyKeyboardMarkup
+from telegram.ext import CommandHandler, Updater
+
+
+load_dotenv()
+config = configparser.ConfigParser()
+config.read('setup.cfg', encoding='utf-8')
+locale.setlocale(locale.LC_TIME, 'ru')
+
+
+TELEGRAM_TOKEN = os.getenv('TELEGRAM_TOKEN')
+LOG_LEVEL = config['log_setting']['log_level']
+FORMAT_LOG = config['log_setting']['log_format']
+LOGS_DIRECTORY_PATH = config['log_setting']['logs_directory_path']
+LOG_SIZE = int(config['log_setting']['log_size'])
+BACKUP_СOUNT = int(config['log_setting']['backup_сount'])
+TELEGRAM_ADMIN_ID = config['tg_setting']['admin_tg_id']
+RETRY_TIME = int(config['default']['retry_time'])
+TZ_MOSCOW = pytz.timezone(config['tg_setting']['tzone'])
+DT_MOSCOW = datetime.datetime.now(TZ_MOSCOW)
+ENDPOINTS = []
+CHECK_RESULT = []
+BUTTONS = ReplyKeyboardMarkup(
+    [['/bot_settings', '/last_check', '/subscribe']],
+    resize_keyboard=True,
+)
+
+
+def get_subscribers_ids():
+    """Достаёт из файла setup.cfg перечень id учётных запписей телеграм."""
+    logger.info('***Работает get_subscribers_ids.')
+    telegram_subscriber_ids = []
+    tg_ids = config['tg_setting']['subscription_tg_ids'].split(',')
+    for tg_id in tg_ids:
+        if isinstance(int(tg_id), int):
+            telegram_subscriber_ids.append(int(tg_id))
+    logger.debug(
+        'Список ИД учёток Телеграм подписчиков из get_subscribers_ids - '
+        + f'{telegram_subscriber_ids}.')
+    return telegram_subscriber_ids
+
+
+def send_message_admin(bot, message):
+    """Отправляет сообщение в Telegram чат админа."""
+    logger.info('***Работает send_message_admin.')
+    if not bot:
+        message = f'Ошибка инициализации объекта bot - {bot}.'
+        logger.error(message, exc_info=True)
+        raise MyCustomError(message)
+    else:
+        bot.send_message(chat_id=TELEGRAM_ADMIN_ID, text=message,)
+        logger.info(f'Админу отправлено сообщение - "{message}".')
+
+
+def send_message(bot, message, telegram_ids):
+    """Отправляет сообщение в Telegram-чаты участников процесса."""
+    logger.info('***Работает send_message.')
+    if not bot:
+        message = f'Ошибка инициализации объекта bot - {bot}.'
+        logger.error(message, exc_info=True)
+        raise MyCustomError(message)
+    else:
```

Comment:
я бы здесь переписал без if и без передачи бота в параметры (бот же один, зачем его здесь прокидывать)
```suggestion
BOT = Bot(token=TELEGRAM_TOKEN)

def get_bot():
    if not BOT:
        message = f'Ошибка инициализации объекта bot - {bot}.'
        logger.error(message, exc_info=True)
        raise MyCustomError(message)
    return BOT

def send_message(message, telegram_ids):
    """Отправляет сообщение в Telegram-чаты участников процесса."""
    bot = get_bot()
```
И этот метод можно будет использовать в send_message_admin аналогично